OpenCL_C.txt

```diff
@@ -2482,94 +2737,102 @@ p = cp; // illegal
 // it is illegal to convert from a generic pointer
 // to an explicit address space pointer without a cast:
 gp = p; // compile-time error
-lp = p; // compile-time error
+lp = pi; // compile-time error
```

Comment:
Was this change intended?  I believe this should remain `p` instead of `pi`.

Reply:
Yes, this is unintentional! Thanks!